test/libsolidity/syntaxTests/userDefinedValueType/explicit_conversions.sol

```diff
@@ -0,0 +1,10 @@
+type MyUint is uint;
+type MyAddress is address;
+
+function f() pure {
+    // TODO Should we allow explicit conversions from literals?
```

Comment:
I think yes. This is similar to how newtype semantics work in haskell and rust.

Reply:
I'm not too sure about it - wouldn't it create ambiguities? We have to at least check that implicit conversion from the literal to the underlying type is possible.